src/PushCommand.php

```diff
@@ -400,6 +412,56 @@ private function getPackageName(InputInterface $input = null)
         }
     }
 
+    /**
+     * @param InputInterface $input
+     */
+    private function parseNexusExtra(InputInterface $input){
+        try{
+            $this->checkNexusPushValid($input);
+
+            $repository = $input->getOption(self::REPOSITORY);
+            $extras = $this->getComposer(true)->getPackage()->getExtra();
+            if(empty($repository)){
```

Comment:
When the `repository` parameter is not set, we should use the first one of the list as default if we have a list. Not blocker but can be improved for the future.

Reply:
I don't recommend adding this strategy at the moment.  First of all, the current version is compatible with former unique repository configuration format. With this parameter, users can clearly know the difference between the two modes, so as to avoid using mistakes. In addition, it can ensure that in the future, users inadvertently adjust the order of the warehouse list, the program execution results exceed expectations, and components are not uploaded to expected nexus repository

Reply:
Fallback to the default is something that I would expect, but I understand your point.